### PR TITLE
Add recipe for pytest-mpl

### PR DESCRIPTION
**Pull request by dopplershift**
This is a plugin for pytest that facilitates tests that check figure
generation.

### CODE REVIEW

File: recipes/pytest-mpl/.cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (pelson) [inline recipes/pytest-mpl/.cache/v/cache/lastfailed]: Is this intended to be here?

**Comment by pelson** [comment]
:+1: with the .cache file question resolved.

**Comment by dopplershift** [comment]
Oops. Removed and rebased.

**Comment by pelson** [comment]
Thanks @dopplershift

**Pull request merged**